codegen-compiler-test/src/test/kotlin/com/freeletics/khonshu/codegen/codegen/FileGeneratorTestComposeFragment.kt

```diff
@@ -185,7 +185,7 @@ internal class FileGeneratorTestComposeFragment {
 
         """.trimIndent()
 
-        test(data, "com/test/Test.kt", source, expected)
+        test(data, "com/test/FragmentTest.kt", source, expected)
```

Comment:
This one should stay as `Test.kt` as it's the file name for the sources, not for the generated code. Same for the other tests below.

Reply:
Ah, we're computing the generated file name based on this as well https://github.com/freeletics/khonshu/blob/main/codegen-compiler/src/testFixtures/kotlin/com/freeletics/khonshu/codegen/KhonshuCompilation.kt#L154-L158. So let's keep it as is